package.json

```diff
@@ -1,39 +1,46 @@
 {
   "name": "tachyon",
-  "version": "2.6.1",
+  "version": "3.0.0",
+  "type": "module",
   "repository": {
     "type": "git",
     "url": "https://github.com/humanmade/tachyon.git"
   },
   "description": "Human Made Tachyon in node",
-  "main": "index.js",
+  "main": "dist/lambda-handler.js",
   "config": {
     "bucket": "",
     "path": "",
     "region": "us-east-1",
     "function-name": ""
   },
   "scripts": {
-    "build-node-modules": "rm -R node_modules ; docker run --rm -v `pwd`:/var/task public.ecr.aws/sam/build-nodejs14.x:latest npm install",
-    "test-file": "docker run --rm -e S3_BUCKET=hmn-uploads-eu -e S3_REGION=eu-west-1 -v `pwd`:/var/task public.ecr.aws/lambda/nodejs:14 \"lambda-handler.handler '{\"path\":\"/'$npm_config_path'\", \"headers\":{}}'\"",
-    "test": "docker run --rm -v `pwd`:/var/task -it --entrypoint='node' public.ecr.aws/lambda/nodejs:14 /var/task/test-filesize/index.js",
-    "update-test-fixtures": "docker run --rm -v `pwd`:/var/task -it --entrypoint='node' public.ecr.aws/lambda/nodejs:14 /var/task/test-filesize/index.js --update-fixtures",
-    "build-zip": "rm lambda.zip; zip -r --exclude='node_modules/aws-sdk/*' --exclude='node_modules/animated-gif-detector/test/*' lambda.zip ./node_modules/ index.js proxy-file.js lambda-handler.js",
+    "build": "sam build -u",
```

Comment:
Is `sam` actually added as a dependency? (Is it in aws-lambda?)

Reply:
No, it's part of https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/install-sam-cli.html. I think we need to document it in CONTRIBUTING.md though

Reply:
OK documented